drivers/SmartThings/matter-sensor/profiles/temperature-humidity-battery.yml

```diff
@@ -19,3 +19,16 @@ preferences:
     explicit: true
   - preferenceId: humidityOffset
     explicit: true
+config:
```

Comment:
The root level property name has to be `deviceConfig`. Also, until [CODA-3870](https://smartthings.atlassian.net/browse/CODA-3870) has been implemented, the device config has to contain all of the sections, i.e. `detailView` and `automation`. Same comment for the other files